dask/array/tests/test_fft.py

```diff
@@ -108,3 +113,41 @@ def test_fft_consistent_names():
     assert same_keys(fft(darr, 5), fft(darr, 5))
     assert same_keys(fft(darr2, 5, axis=0), fft(darr2, 5, axis=0))
     assert not same_keys(fft(darr, 5), fft(darr, 13))
+
+
+def test_wrap_numpy():
```

Comment:
IANAL. That said, my understanding was as long as the GPL dependencies are optional and are not shipped with the distribution, this should not effect licensing. This SO [answer]( http://softwareengineering.stackexchange.com/a/175905 ) seems pertinent.

Note: FFTW is GPL 2.